client/src/unifycr-sysio.c

```diff
@@ -790,6 +790,44 @@ int UNIFYCR_WRAP(open64)(const char *path, int flags, ...)
     return ret;
 }
 
+int UNIFYCR_WRAP(__open_2)(const char *path, int flags, ...)
+{
+    DEBUG("__open_2 was called for fd %d....\n", fd);
```

Comment:
No fd defined here, mean to report the path name?

Reply:
@adammoody thanks. yeah i copied that debug statement from another wrapper and missed that.

Reply:
@adammoody I updated/fixed the debug statement here.